src/test/java/org/alfresco/repo/search/impl/querymodel/impl/db/DBQueryTest.java

```diff
@@ -268,8 +264,9 @@ protected void createTestData() throws Exception
         I18NUtil.setLocale(Locale.UK);
         txn = transactionService.getUserTransaction();
         txn.begin();
-        this.authenticationComponent.setSystemUserAsCurrentUser();
-        
+//        this.authenticationComponent.setSystemUserAsCurrentUser();
+        this.authenticationComponent.setCurrentUser("admin");

```

Comment:
Why are we setting the current user as admin? What was the issue?

Reply:
My mistake. I have put it back as before.